package.json

```diff
@@ -39,25 +39,30 @@
     "webpack": "2.4.1"
   },
   "dependencies": {
+    "@tinymce/tinymce-react": "^3.2.0",
```

Comment:
nit: I suggest pegging the version exactly - we've had problems (especially in consent-ui) with the compatible version annotations that pulled in updates that were actually incompatible. In other words, I don't trust NPM versioning promises - they have caused huge headaches in the past.